app/concepts/matestack/ui/core/app/app.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue/dist/vue.esm'
-import axios from 'axios'
```

Comment:
wasn't used anywhere, in the file so I removed it. I hope I'm not unaware of some side effect this had, but the tests pass :man_shrugging: